django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -1,5 +1,10 @@
 (function ($) {
+    var toggleFullScreen = function(){
+       $('.field-config').toggleClass('full-screen');
+    }
+
     var initAdvancedEditor = function(target, data, schema){
+
```

Comment:
remove blank line for consistency with the rest of the code (increase line-height in your editor if you think lines are too close together)